suites/rbd/qemu/pool/ec-data-pool.yaml

```diff
@@ -0,0 +1,21 @@
+tasks:
+- exec:
+    client.0:
+      - sudo ceph osd erasure-code-profile set teuthologyprofile ruleset-failure-domain=osd m=1 k=2
+      - sudo ceph osd pool delete rbd rbd --yes-i-really-really-mean-it
+      - sudo ceph osd pool create rbd 4 4 erasure teuthologyprofile
```

Comment:
I missed the fact that the librbd and cli tests did the cache tier as well -- in that case, I would say yes, collapse them down into a "pool" configuration that tests none, replicated data pool, ec data pool, small replicated cache tier